spec/middleman-prismic/preview_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "spec_helper"
+require "middleman-prismic/preview"
+
+describe Middleman::Prismic::Preview do
+  it "passes through requests outside of /preview" do
+    app = rack_app
+    allow(Kernel).to receive(:system)
+    env = stub_env("http://example.com/")
+
+    middleware = Middleman::Prismic::Preview.new(app)
+    code, env, message = middleware.call env
```

Comment:
Since we're not using `env` after this line does it make sense not to re-assign the variable and instead do something like:

```rb
code, _env, message = middleware.call env
```

Reply:
I think `env` is a misleading name here, how about `headers` or `response_headers`?